app/controllers/games_controller.rb

```diff
@@ -1,4 +1,33 @@
 class GamesController < ApplicationController
-  # Add your GamesController code here
+  before_action :find_game, only: [:show, :update]
+
+  def create #POST
+    game = Game.create(game_params)
+    render json: game
+  end
+
+  def show #GET /games/:id
+    render json: @game
+  end
+
+  def update #PATCH /games/:id
+    @game.update(game_params)
+    render json: @game
+  end
+
+  def index #Get /games
+    games = Game.all
+    render json: games
+  end
+
+private
+
+  def game_params
+    params.permit(state: [])
+  end
+
+  def find_game
+    @game = Game.find(params[:id])
+  end
 
```

Comment:
You should always ensure that you are whitelisting your parameters to only allow approved params to be used in the controller.

```ruby
class GamesController < ApplicationController
  #your game routes

  private
    def game_params
      params.permit(state: [])
    end
end
```